### ISSUE TITLE
Question: animated emoji sources

### ISSUE THREAD

**Issue by Sominemo** [microsoft/fluentui-emoji#12]
Some emojis in Microsoft Teams are animated. Are there any plans to provide assets/sources for these?

**Comment by wukko** [comment]
there's a cover for animation in [figma community file](https://www.figma.com/community/file/1138254942249677742), so maybe they are/were planning on doing it 

![image](https://user-images.githubusercontent.com/71202418/184019004-3503d13b-ba7f-4be1-891a-a6334602923f.png)

**Comment by jasoncuster** [comment]
hahaha, welp...I should definitely break the link so you don't see all our covers. Good catch. 

There is a file, but it's just stitching the png sequence together with delayed timing for prototyping. Currently we're not sharing the animations yet, but hopefully that will change in the future. 👀

**Comment by Maxou44** [comment]
Any news about animated emojis ? 😏

**Comment by tonygaglio** [comment]
^ also interested in this - we would love if these become available one day 🙏

**Comment by MichaelPriebe** [comment]
Posted some info if people want to get the animated emoji: https://github.com/microsoft/fluentui-emoji/issues/51#issuecomment-1786429695

**Comment by Not-Qualified** [comment]
> Posted some info if people want to get the animated emoji: [#51 (comment)](https://github.com/microsoft/fluentui-emoji/issues/51#issuecomment-1786429695)

https://em-content.zobj.net/source/microsoft-teams/363/fire_1f525.png -> Here 256 x 256 resolution is available

**Comment by MichaelPriebe** [comment]
> > Posted some info if people want to get the animated emoji: [#51 (comment)](https://github.com/microsoft/fluentui-emoji/issues/51#issuecomment-1786429695)
> 
> https://em-content.zobj.net/source/microsoft-teams/363/fire_1f525.png -> Here 256 x 256 resolution is available

These are from Emojipedia, and they are missing a few, like woman pilot, and bandaid.

**Comment by Not-Qualified** [comment]
@MichaelPriebe , Yes, They are missing some, yet you will going to have bigger resolution available also you can go to Microsoft Teams and can find source from their if you reverse engineer and also look for https://www.emojiall.com/ because they too have big emoji repository of their own
